testing/trino-benchto-benchmarks/src/test/resources/sql/presto/tpcds/q02.plan.txt

```diff
@@ -9,14 +9,18 @@ remote exchange (GATHER, SINGLE, [])
                                 partial aggregation over (d_week_seq)
                                     join (INNER, REPLICATED):
                                         remote exchange (REPARTITION, ROUND_ROBIN, [])
-                                            scan web_sales
-                                            scan catalog_sales
+                                            filter "@$internal$dynamic_filter_function@52QIVVB4140BQ100UB41O6JGAS6H8D16SHJ2H9BMT9LA24OGG790BO798805A3VUID8PP6L4EL82LMQC6QFI9P0JUMKHAK0D32Q21KAJIF0MGO4CGT9TFASB49BQP0RQMDKGM8P6SCSCG4OLIPDIF7TQEMQDIDN5S1M4D5C3D61HM984GF5TJAU4LN5GDTT4EN4QQS8NGL70K3007K941V1NL1SBQ1375CEPTI6A7Q8DLQG5BE7F235E6DLA109I0E478USL"("ws_sold_date_sk", 'EQUAL', 'df_2177', false)
```

Comment:
Could we get rid of this long hash code? You can use `io.trino.sql.DynamicFilters#extractDynamicFilters` to do so